src/components/PageTitle.vue

```diff
@@ -1,26 +1,31 @@
 <template>
-  <div class="mb-5" :class="{ 'border-b border-primary-40 pb-3': border }">
-    <div class="flex justify-between items-center">
+  <div>
+    <div
+      class="flex justify-between items-center"
+      :class="{ 'has-border border-b border-primary-40 pb-3 mb-5': border }"
+    >
       <AppTitle
+        light
         class="mr-4 text-primary-80"
         :class="{ 'hidden md:block': !noCollapse }"
       >
         {{ title }}
       </AppTitle>
       <slot></slot>
     </div>
-    <div v-if="subTitle" class="text-sm text-primary-80">
-      {{ subTitle }}
-    </div>
   </div>
 </template>
 <script lang="ts" setup>
 import AppTitle from './AppTitle.vue';
 
 defineProps<{
   title: string;
-  subTitle?: string;
   border?: boolean;
   noCollapse?: boolean;
 }>();
 </script>
+<style lang="postcss" scoped>
+.has-border h1 {
```

Comment:
Whenever you want to create classes (especially ones with non-class-only selectors) there is probably a more "Tailwindy" way. In this case it's actually much simpler just to remove `pb-3` from the border then you don't need to do anything to AppTitle